ui/src/app/store/types/model.ts

```diff
@@ -2,6 +2,13 @@ export type Model = {
   id: number;
 };
 
+export type TimestampFields = {
+  created: string;
+  updated: string;
+};
+
+export type TimestampedModel = Model & TimestampFields;
```

Comment:
Personally I think this _might_ be complicating things unnecessarily. I might be clearer if the model was left as is and we treated time stamp fields separately without creating a separate type that's intersection of the two.

The naming makes sense. Maybe it's just a matter of getting used to it. 🤷‍♂️